os/fs/smartfs/smartfs_utils.c

```diff
@@ -2169,7 +2169,7 @@ int smartfs_sector_recovery(struct smartfs_mountpt_s *fs)
 	/* TODO Find all active Logical sectors from root sector and unmark for exist sector */
 	ret = smartfs_scan_entry(fs, map, &info);
 	if (ret != OK) {
-		fdbg("BUG!! scan entry failed...\n");
+		fdbg("BUG!! scan entry failed, ret : %d\n", ret);
```

Comment:
Since you are making this PR to clean up the error logs, i have one suggestion. 
The error logs are having different formats in all these files. Some places it is "Error during xxx.. "  or "Failed to xxx.." or "Unable to xxx...". Here it is "BUG1! ". I think it is better to have a standard format or prefix for all error logs. For example, all error logs must start with "ERROR: ". The other things like "Failed to " and "Unable to " are actually part of the body of the error message. Having a standard format will help in searching and identifying error logs easily. This will be especially useful when we have large log dumps.

Reply:
Agreed,
Trying to align the Error debug messages,
For calls to other fs layer functions, will keep the message as follows: -
`fdbg("Unable to xxx, ret : %d", ret);
`

For calls to MTD layer IOCTLs, will keep the message as follows: -
`fdbg("Error in xxx,  ret : %d", ret);
`

This helps in demarcating fs layer errors and MTD layer errors too,
while keeping the style as suggested by Taejun.
Your opinion please.